107/src/main/java/org/ehcache/jsr107/Eh107MutableConfiguration.java

```diff
@@ -36,4 +52,20 @@ public Eh107MutableConfiguration(CompleteConfiguration<K, V> configuration) {
   public boolean isExpiryPolicyOverridden() {
     return this.expiryPolicyOverridden;
   }
+  
```

Comment:
Don't fix others' mess in your PR, risk making things harder if their fix ends up going first.